### DISCUSSION TITLE
X and Y axis motors heating

### DISCUSSION THREAD

**Discussion by Rosimar-Lima** [Jyers/Marlin#1369] [General Discussion]
I bought a creality v4.2.2 board with A4988 drives and a 4.3 inch display (not touchscreen). I installed firmware E3V2-ManualMesh-3x3-v4.2.2-v1.3.5.b.
I noticed that the x- and y-axis motors are heating up to a reasonably high temperature. I would like to know if there is any parameter in the firmware that determines the ideal current for these motors or if it could be the driver with high Vref.

**Comment by Tapap84** [comment]
you need to change your vref. thats what i see heat up motors the most is too high vref. so many things go into this. 2 boards i ordered had vref set so bad that im surprised stuff didnt burn

this is taken from a vref cacl site i use. do your own research first.

assuming you have stock motors which are 1.5a
red Ω r050 vref is 0.60
red Ω r068 vref is 0.81
green Ω r100 vref is 1.2
red Ω r200 vref is 2.4

**Reply by Tapap84** [reply]
https://mak3r.de/2020/04/03/stepper-driver-current-and-vref-calculator/#1594211222021-3d3d72ae-0ff4
